src/main/java/uk/gov/pay/api/resources/PaymentsResource.java

```diff
@@ -241,7 +247,11 @@ public Response searchPayments(@ApiParam(value = "accountId", hidden = true)
             @ApiResponse(code = 429, message = "Too many requests", response = ApiErrorResponse.class),
             @ApiResponse(code = 500, message = "Downstream system error", response = PaymentError.class)})
     public Response createNewPayment(@ApiParam(value = "accountId", hidden = true) @Auth Account account,
-                                     @ApiParam(value = "requestPayload", required = true) ValidCreatePaymentRequest validCreatePaymentRequest) {
+                                     @ApiParam(value = "requestPayload", required = true) CreatePaymentRequest createPaymentRequest) {
+        logger.info("Payment create request parsed to {}", createPaymentRequest);
+
+        paymentRequestValidator.validate(createPaymentRequest);
+        ValidCreatePaymentRequest validCreatePaymentRequest = new ValidCreatePaymentRequest(createPaymentRequest);
         logger.info("Payment create request passed validation and parsed to {}", validCreatePaymentRequest);
```

Comment:
It’s possibly a bit redundant to include the `validCreatePaymentRequest` as now we have logging a few lines above that outputs the `createPaymentRequest`.

Reply:
I kept it temporarily whilst during the transition so if anything goes wrong its easier to spot whether it occurred within the custom deserialisation or the custom validation class. I agree that once we're done we should remove it to one log line.